packages/react-cap-theme/stories/components/Button.stories.tsx

```diff
@@ -9,70 +9,172 @@ import {
+      hoverButton?.classList.add('pseudo-hover');
+
+      const activeButton = doc.getElementById(stateId(appearance, 'active'));
+      activeButton?.classList.add('pseudo-hover', 'pseudo-active');
+
+      const focusButton = doc.getElementById(stateId(appearance, 'focus'));
+      if (focusButton) {
+        focusButton.classList.add('pseudo-focus-visible');
+        focusButton.setAttribute('data-fui-focus-visible', 'true');
+      }
+    });
+  }, [selectedTheme, iconPosition, shape, size, disabledFocusable]);
+
   return (
     <CAPThemeExamples
       examples={[
         {
-          title: 'Default',
-          render() {
-            return <Button icon={<CalendarMonthRegular />}>Default</Button>;
-          },
-        },
-        {
-          title: 'Primary',
+          title: 'States Overview',
           render() {
             return (
-              <Button appearance="primary" icon={<CalendarMonthRegular />}>
-                Primary
-              </Button>
-            );
-          },
-        },
-        {
-          title: 'Outline',
-          render() {
-            return (
-              <Button appearance="outline" icon={<CalendarMonth />}>
-                Outline
-              </Button>
-            );
-          },
-        },
-        {
-          title: 'Subtle',
-          render() {
-            return (
-              <Button appearance="subtle" icon={<CalendarMonth />}>
-                Subtle
-              </Button>
-            );
-          },
-        },
-        {
-          title: 'Transparent',
-          render() {
-            return (
-              <Button appearance="transparent" icon={<CalendarMonth />}>
-                Transparent
-              </Button>
-            );
-          },
-        },
-        {
-          title: 'Tint',
-          render(variant) {
-            if (variant === 'v9') {
-              return 'NOT_IMPLEMENTED';
-            }
-            return (
-              <Button appearance="tint" icon={<CalendarMonth />}>
-                Tint
-              </Button>
+              <div style={{ display: 'flex', flexDirection: 'column', gap: '32px', padding: '32px 24px' }}>
+                {buttonAppearances.map((appearance) => {
+                  return (
+                    <div
+                      key={appearance}
+                      style={{
+                        display: 'flex',
+                        alignItems: 'center',
+                        gap: '24px',
+                      }}
+                    >
+                      <div style={{ fontWeight: 600, textTransform: 'capitalize', width: '100px', textAlign: 'right', margin: '0 24px 0 0' }}>
+                        {appearance}
+                      </div>
+                      <Button
+                        id={stateId(appearance, 'default')}
+                        icon={<CalendarMonth />}
+                        {...commonButtonProps}
+                        appearance={appearance}
```

Comment:
could you also add `@ts-expect-error` suppression here? RIT tests are failing on CI because there is no `tint` appearance variant in official Button API 

<img width="2514" height="1356" alt="image" src="https://github.com/user-attachments/assets/b69f3844-cc3f-4c60-b1f3-f4583e752dfe" />